Task_DEV_3/Task_DEV_3/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace Task_DEV_3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the size of the array :");
+            int arrayLength = Convert.ToInt32(Console.ReadLine());
+            if (arrayLength <= 5)
+            {
+                int[] inputNumbers = new int[arrayLength];
+
+                for (int i = 0; i < arrayLength; i++)
+                {
+                    Console.WriteLine($"Enter an array element № {i}");
+                    inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
+                }
+
+                bool isPrimeNumber = true;
+
+                for (int i = 0; i < arrayLength; i++)
+                {
+                    for (int j = 2; j < inputNumbers[i]; j++)
+                    {
+                        if (inputNumbers[i] % j == 0)
+                        {
+                            isPrimeNumber = false;
+                            break;
+                        }
+                    }
+                    if (isPrimeNumber == false)
```

Comment:
Why do you use inverse logic here? It would be more preferable to use forward logic. Smth like` if (isPrimeNumber)...else`

Reply:
Done